src/api/movie_api.ts

```diff
@@ -0,0 +1,145 @@
+import {Filters} from 'src/utils/filters'
+import axios from 'axios'
+import {useInfiniteQuery, UseInfiniteQueryResult, useQuery} from 'react-query'
+
+const apiURL = import.meta.env.VITE_API_URL
+const apiKEY = import.meta.env.VITE_API_KEY
+const TORRENT_API_URL = 'https://yts.lt/api/v2/'
+
+export function client(
+  url: string,
+  params?: Record<string, string | number | boolean>,
+): Promise<any> {
```

Comment:
```suggestion
): Promise<unknown> {
```

I suggest typing it to `unknown`. `any` is a foot gun.